src/extension/module.js

```diff
@@ -418,6 +418,7 @@
       glob = '**';
     }
     const reString = glob
+      .replace('.', '//.') // don't match every char, just real dots
```

Comment:
Shouldn't it be like this?
```suggestion
      .replace('.', '\\.') // don't match every char, just real dots
```